### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by kiudee**
## Description
This pull request is used to prepare the first minor release from 1.0 -> 1.1.
It updates the `HISTORY.rst` file with a changelog and adds classifiers to the setup.py file.
These classifiers are useful for filtering packages on PyPI.

## Does this close/impact existing issues?
It "closes" milestone 1.1.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -46,12 +59,9 @@
                 "h5py>=2.7",
                 "pygmo>=2.7",
             ],
-            "probabilistic": [
-                "pymc3>=3.8",
-                "theano>=1.0",
-            ],
+            "probabilistic": ["pymc3>=3.8", "theano>=1.0",],
```
Comment (timokau) [inline setup.py]: My only nitpick is that I would prefer to keep unrelated formatting changes out of functional commits (after the release would probably be a good time to continue with #78). But that's not terribly important.

Otherwise looks good to me :)

Comment (kiudee) [reply setup.py]: I agree that this can obfuscate the true changes of the commit. I should turn off my auto-formatting on save.
